uast/uast.go

```diff
@@ -28,11 +28,11 @@ type Hash uint32
 type Role int16
 
 const (
-	_ = iota
+	Invalid Role = iota
```

Comment:
Maybe a little comment like:

```go
// Protobuf enums start with 0 so we assign it the Invalid Role
```

or something like that so whoever see the code understand the need for this.